boards/chronos/include/periph_conf.h

```diff
@@ -26,9 +26,16 @@ extern "C" {
 #endif
 
 /**
- * @brief   Clock configuration
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```

Comment:
The problem is that `@name` names the group, but TODOs for the group need to be in the group ;-)